src/firefly/test/edu/caltech/ipac/visualize/plot/ZscaleTest.java

```diff
@@ -14,58 +16,58 @@
  * Created by ymei on 3/28/16.
  * 10/19/16
  *  DM-8028
- *    Use teh UnitTestUtility to load file
+ *    Use the UnitTestUtility to load file
+ * 01/07/2017 DM-7188
+ *    Added the setUp() and tearDown(); Directly called Zscale.cdl_zscale;
  */
 public class ZscaleTest {
 
+    //Input fits file:
     private static String filename = "WISE-Band-4.fits";
+
+    //Prepare the parameters for calling cdl_zscale:
+    private ImageHeader imageHeader = null;
+    private float[] float1d = null;
+
     
-/**
-    @Test
-    public void testFitLineRetvalAbnormalInputs() {
-        float[] data = {1,2,3,4,5,6,7,8,9};
-        int npix = 0;
-        float krej = 2.2f;
-        int ngrow = 6;
-        int maxiter = 5;
-        Zscale.FitLineRetval fitLineRetval = Zscale.fitLine (data, npix, krej, ngrow, maxiter);
+    @Before
+    /**
+     * An one dimensional array is created and it is used to run the unit test for Histogram's public methods
+     */
+    public void setUp() throws FitsException {
 
+        Fits fits = FileLoader.loadFits(ZscaleTest.class,filename );
+        FitsRead[] fry = FitsRead.createFitsReadArray(fits);
+        float1d = fry[0].getDataFloat();
+        imageHeader = fry[0].getImageHeader();
+
+    }
 
-    // Private!!!
+    @After
+    /**
+     * Release the memories
+     */
+    public void tearDown() {
+        float1d = null;
+        imageHeader = null;
     }
```

Comment:
Everything looks great. The unit test class is clean and well structured. It runs well.
You can merge it now.